power_system_fault.py

```diff
@@ -73,3 +79,61 @@ def phase_current_b(self):
 
     def phase_current_c(self):
         return self.phase_current_a()
+
+
+class LineToLineFault(Fault):
+    def sequence_current_0(self):
+        return 0j
+
+    def sequence_current_1(self):
+        # we should have this in _init
+        n = self._fault_bus - 1
+        v_f = self._system.buses[n].voltage
+        z_0 = self._impedance_matrix_0[n][n]
+        z_1 = self._impedance_matrix_1[n][n]
+        z_2 = self._impedance_matrix_2[n][n]
+        return v_f / (z_1 + z_2 + self._fault_impedance)
+
+    def sequence_current_2(self):
+        return -1 * self.sequence_current_1()
+
+    def phase_current_a(self):
+        return 0j
+
+    def phase_current_b(self):
+        return (A**2 - A) * self.sequence_current_1()
+
+    def phase_current_c(self):
+        return -1 * self.phase_current_b()
+
+
+class DoubleLineToGroundFault(Fault):
+    def sequence_current_0(self):
+        return (-1*self.sequence_current_1()) - self.sequence_current_2()
```

Comment:
Formatting (spaces around the *). Use Ctrl-Alt-L and PyCharm will autoformat the file to PEP-8 standards.